modules/apps/powwow/powwow-portlet/docroot/WEB-INF/src/com/liferay/powwow/provider/zoom/ZoomPowwowServiceProvider.java

```diff
@@ -325,7 +338,8 @@ protected JSONObject execute(
 		PowwowServer powwowServer, List<String> resourceParams, Http.Method method,
 		Map<String, String> parameterMap) {
 
-		return execute(powwowServer, resourceParams, Http.Method.GET, null, true);
+		method = Validator.isNull(method) ? Http.Method.GET : method;
```

Comment:
do not need this check. by default the method `execute` already uses method `GET`